### PR TITLE
Assets work

### LINKED ISSUE #2231: Some UIs issues using baggy
It looks like we have some issues regarding the baggy theme on the 2.1.
Here is a preview on my local machine:

image

- [ ] Titles have padding on the left
- [x] Filter panel is open by default and I can't close it
- [ ] Clear button doesn't work

### PR DESCRIPTION
**Pull request by tcitworld**
| Q             | A
| ------------- | ---
| Bug fix?      | yes and no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Documentation | no
| Translation   | no
| Fixed tickets | #2231, #2249
| License       | MIT

* ES6 all the things ! (Modules !)

* ESLint checks everything

* CSS fixes (use stylelint)

* Fix #2231

### CODE REVIEW

**Comment by tcitworld** [comment]
Please test many use cases. :-)

File: .gitignore [hunk 1]
```diff
@@ -49,3 +49,7 @@ app/Resources/static/lib/*
 /bin
 !/src/Wallabag/CoreBundle/Resources/public
 /src/Wallabag/CoreBundle/Resources/public/*
+
+# PHPStorm
+.idea/
```
Comment (nicosomb) [inline .gitignore]: you have to put these lines in a global gitignore

**Comment by tcitworld** [comment]
@j0k3r I can't reproduce. Did you forgot to run `grunt` after checkout ?

**Comment by j0k3r** [comment]
Ok it works again, maybe I forgot to run `grunt`.

Anyway,
- the panel can't be close
- clear works great

**Comment by tcitworld** [comment]
> the panel can't be close

I can on my side. Maybe clear wallabag & browser cache ?

**Comment by j0k3r** [comment]
I still can't.
Could you point where is the function which should close it?

**Comment by tcitworld** [comment]
It should be https://github.com/wallabag/wallabag/pull/2314/files#diff-b6cebb35827533b53fb7f26090c13f6bR254

**Comment by tcitworld** [comment]
@j0k3r Should work now.

- Font ligatures are not working

**Comment by j0k3r** [comment]
So we can ditch bower? :laughing:

**Comment by tcitworld** [comment]
- [ ] Fix for twitter icons becoming facebook one (WTF ?)

File: package.json [hunk 2]
```diff
@@ -10,6 +10,8 @@
   "dependencies": {
     "annotator": "git://github.com/wallabag/annotator.git#0f076c7d371ed25eb0793346f46982d90f2c4c85",
     "autoprefixer": "^6.3.6",
+    "babel-preset-es2015": "^6.14.0",
+    "babelify": "^7.3.0",
     "bower": "^1.7.7",
```
Comment (j0k3r) [inline package.json]: Can be removed too

**Comment by nicosomb** [comment]
👍

**Pull request merged**